app/views/items/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<html>
+  <body>
+    <div class="pageheaders">
+      <h1>New Item</h1>
+    </div>
+
+    <div class="form">
+      <%= form_tag("/merchants/#{@merchant.id}/items", method: :post) do %>
+        <p>Enter a new item name:</p>
+        <%= text_field_tag :name %>
+        <p>Enter description:</p>
+        <%= text_field_tag :description %>
+        <p>Enter price:</p>
+        <%= text_field_tag :price %>
+        <p>Enter product image url:</p>
+        <%= text_field_tag :image %>
+        <p>Enter inventory amount:</p>
+        <%= text_field_tag :inventory %><br><br>
+        <%= hidden_field_tag :status, "active" %>
+        <%= hidden_field_tag :merchant_id, @merchant.id %>
+        <%= submit_tag "Create Item" %>
+      <% end %>
+    </div>
+  </body>
+</html>
```

Comment:
Just a general comment for all of your views... All of your views are getting yielded within the application.html.erb file. In this file, there is the basic structure for your page layout, and whatever you put in these views are rendered into the body. So you don't need to include <html> and <body> tags since it will automatically go into the body element of the page.